### DISCUSSION TITLE
破解台北都市傳說！深夜一輛公車開進信義區，卻沒人握著方向盤？

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#712] [General]
https://futurecity.cw.com.tw/article/1603
不再遇到態度不好的司機了，ＡＩ絕對不會對你不理不睬

**Comment by jerry0108566** [comment]
第一次聽到這輛公車的時候還是在高中，半夜都能見到他在信義路上試駕，也不知道如今開發到什麼程度了，啥時能夠正式營運。

**Comment by alvin1226** [comment]
這樣公車司機是不是又要失業了QQ

**Comment by HsuChen88** [comment]
幽靈公車，付雙倍車資，不然無法下車。
